components/__design__/text.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import '/styles/new/base.css'; // eslint-disable-line
```

Comment:
There's no need to import this here, you probably want to add `@import 'base.css'` within `styles/new/index.css` before the `@tailwind` statements.

Reply:
ah thank you, that's exactly what I wanted to do. made the changes :)